rest/plugin.spec.yaml

```diff
@@ -4,7 +4,7 @@ products: [insightconnect]
 name: rest
 title: REST
 description: The REST plugin to make it easy to integrate with RESTful services
-version: 3.0.1
+version: 4.0.0
```

Comment:
No underlying change for the customer and is backwards compatible as a bugfix, so would just set to `3.0.2`